File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Super_Easy/12577-Hajj_e_Akbar/Main.java
/*
 * Problem 12577 - Hajj-e-Akbar
 * Java version 6/10/2016
 */

import java.io.PrintStream;
import java.util.Scanner;

public class Main {
    static final Scanner input = new Scanner(System.in);
    static final PrintStream output = System.out;

    public static void main(String[] args) {

        StringBuffer results = new StringBuffer();

        int i = 1;
        while (true) {
            String s = input.next();
            if (s.equals("*")) {
                break;
            }

            results.append("Case " + i++ + ": ");
            if (s.equals("Hajj")) {
                results.append("Hajj-e-Akbar\n");
            } else {
                results.append("Hajj-e-Asghar\n");
            }
        }
        System.out.print(results.toString());
    }
}

File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Super_Easy/12577-Hajj_e_Akbar/main.cpp
/*
 * UVa 12577 - Hajj-e-Akbar
 * Straightforward
 */
#include <iostream>
#include <sstream>

int main()
{
  std::ios_base::sync_with_stdio(false);
  std::cin.tie(nullptr);

  std::ostringstream output;
  std::string input {};
  int test {1};
  while (std::cin >> input && input != "*") {
    output << "Case " << test++ << ": Hajj-e-"
      << (input == "Hajj" ? "Akbar\n" : "Asghar\n");
  }
  std::cout << output.str();
}
